recipes/cytoolz/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.7.5" %}
+
+package:
+  name: cytoolz
+  version: {{ version }}
+
+source:
+  fn: cytoolz-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/cytoolz/cytoolz-{{ version }}.tar.gz
+  md5: 15fbc1c56fca80cbca0e89f37c93d432
+
+build:
+  number: 0
+  script: python setup.py install --record=record.txt
```

Comment:
Yeah, I'll be releasing `toolz` and `cytoolz` shortly--as in within a couple hours.

Note that the next version of `cytoolz` will require the next version of `toolz` as a runtime dependency.